test/collection.js

```diff
@@ -1437,6 +1437,17 @@ QUnit.test("#1894 - `sort` can optionally be turned off", function(assert) {
     assert.ok(collection.comparator === comparator);
   });
 
+  QUnit.test("Pass falsey for `models` for empty Col with `options`", function(assert) {
+    assert.expect(6);
+    _.forEach([undefined, null, false], function(falsey) {
+      var MyModel = Backbone.Model.extend();
+      var collection = new Backbone.Collection(falsey, {model: MyModel});
+
+      assert.ok(collection.model === MyModel);
```

Comment:
How about testing the options instead of checking that it attaches properties? (This is currently potentially testing two behaviors).

```js
var opts = {a: 1, b: 2}
_.forEach([undefined, null, false], function(falsey) {
  var Collection = Backbone.Collection.extend({
    initialize: function(options) {
      assert.strictEqual(options, opts);
    }
  });

  var col = new Collection(falsey, opts);
  assert.strictEqual(col.length, 0);
});

```